textattack/shared/utils/install.py

```diff
@@ -122,28 +120,39 @@ def _post_install():
     nltk.download("stopwords")
 
 
+def set_cache_dir(cache_dir):
+    """ Sets all relevant cache directories to ``TA_CACHE_DIR``. """
+    # Tensorflow Hub cache directory
+    os.environ["TFHUB_CACHE_DIR"] = cache_dir
```

Comment:
I'm not sure if this is necessarily good for the user. (1) They might not know that their cache location just got switched. (2) If they already have models and stuff cached, shouldn't we allow them to use it in the first place.

Reply:
resolved w Jeffrey: In the future we'll add multiple levels/preferences for the cache directory